### ISSUE TITLE
Unable to compile on Raspberry Pi

### ISSUE THREAD

**Issue by ChillarAnand** [arduino/ctags#19]
### Describe the problem

```
➜  blink arduino-cli compile --fqbn arduino:avr:mega


Used platform Version Path
arduino:avr   1.8.6   /home/trify/.arduino15/packages/arduino/hardware/avr/1.8.6
Error during build: fork/exec /home/trify/.arduino15/packages/builtin/tools/ctags/5.8-arduino11/ctags: no such file or directory
➜  blink file /home/trify/.arduino15/packages/builtin/tools/ctags/5.8-arduino11/ctags
/home/trify/.arduino15/packages/builtin/tools/ctags/5.8-arduino11/ctags: ELF 64-bit LSB executable, ARM aarch64, version 1 (SYSV), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, BuildID[sha1]=7356c63fb09131d0c66ca5c8249a672ad78f2e8f, not stripped
```

### To reproduce

Install arduino-cli on raspberry pi 4 and compile blink sketch.

### Expected behavior

It should complie sketch.

### Arduino CLI version

Version: 0.34.2 Commit: 963c1a76 Date: 2023-09-11T10:05:41Z

### Operating system

N/A

### Operating system version

➜  blink lsb_release -a No LSB modules are available. Distributor ID:	Raspbian Description:	Raspbian GNU/Linux 11 (bullseye) Release:	11 Codename:	bullseye

### Additional context

➜  blink lsb_release -a
No LSB modules are available.
Distributor ID:	Raspbian
Description:	Raspbian GNU/Linux 11 (bullseye)
Release:	11
Codename:	bullseye

### Issue checklist

- [X] I searched for previous reports in [the issue tracker](https://github.com/arduino/arduino-cli/issues?q=)
- [X] I verified the problem still occurs when using the [nightly build](https://arduino.github.io/arduino-cli/dev/installation/#nightly-builds)
- [X] My report contains all necessary details

**Comment by umbynos** [comment]
The problem here is that we don't have a release distributed for linux-armv7 of arduino ctags.. This will be partially solved by arduino/arduino-cli#2285

**Comment by ChillarAnand** [comment]
Doesn't RPi 4 use linux-armv8(64-bit)?

**Comment by anttiryt** [comment]
> Doesn't RPi 4 use linux-armv8(64-bit)?

True. 32bit os is running in arm-v7 (armhf) compatibility mode even though the processor is arm-v8 capable. 64-bit aarch64 is running in arm-v8.

cat /etc/apt/sources.list 
or
 lsb_release -a

reveal the software version.

**Comment by anttiryt** [comment]
> The problem here is that we don't have a release distributed for linux-armv7 of arduino ctags.. This will be partially solved by [arduino/arduino-cli#2285](https://github.com/arduino/arduino-cli/pull/2285)

@umbynos Is there now a vay to circumvent this issue? I'd like to run arduino-cli on my 32-bit rpi4b

### edit, workaround
This workaround worked for me, as I had already arduino-ctags package installed via apt

```
cd 
cd .arduino15/packages/builtin/tools/ctags/5.8-arduino11
mv ctags orig.ctags
ln -s /usr/bin/arduino-ctags ctags

```
after this build was successful and code ran on nano every.
